### DISCUSSION TITLE
Differentiable Pathloss Parameter

### DISCUSSION THREAD

**Discussion by fanyimeng0** [NVlabs/sionna#588] [General]
Hi, I was wondering if you are interested in developing a differentiable path loss parameter. I am referring to the relationship between path loss and distance.

 I have reviewed **_compute_paths_coefficients** and believe the current exponent is set to 2, but I think it might be meaningful to learn this parameter in certain environments. 

Additionally, do you foresee any potential issues if I set this exponent as a tf.Variable? Thank you!

**Comment by jhoydis** [comment]
Hi @fanyimeng0,

There is no explicit pathloss parameter in Sionna RT. The fact that the power scales as 1/r^2 for LOS and specular paths is because we have spherical wave propagation. (You can check in the [EM Primer](https://nvlabs.github.io/sionna/em_primer.html#) that this is different for paths including diffraction, RIS, or diffuse reflections). This should not be modified and there would be also no physical justification for it as far as I know.

However, I imagine that you would like to have an additional distance-based pathloss, e.g., due to atmospheric absorption, rain, or something like this. One way to hack this into Sionna would be to transform the delay tau_i  for every path into a distance (using the speed of light) and then add an additional pathloss based on this distance.
